docker-compose.yml

```diff
@@ -80,7 +80,7 @@ services:
       --s3.bucket=demobucket
       --s3.prefix.folder=test/output
       --s3.secret.key=secret-key
-      --s3.service.endpoint=http://aws:4572
+      --s3.service.endpoint=http://aws:4566
```

Comment:
why did this change?

Reply:
localstack used to put each service on a different port, s3 was on 4572, they now recommend using a single edge port for all services - so just keeping up to date with that recommendation.